build.sh

```diff
@@ -216,18 +203,19 @@ test_ios_devices() {
         fi
     done <<< "$serial_numbers_str"
     if [[ ${#serial_numbers[@]} == 0 ]]; then
-        echo "At least one iOS device must be connected to this computer to run device tests"
+        echo "At least one iOS/tvOS device must be connected to this computer to run device tests"
         if [ -z "${JENKINS_HOME}" ]; then
             # Don't fail if running locally and there's no device
             exit 0
         fi
         exit 1
     fi
-    scheme="$1"
-    configuration="$2"
-    failed=0
+    local sdk="$1"
+    local scheme="$2"
+    local configuration="$3"
+    local failed=0
     for device in "${serial_numbers[@]}"; do
-        xc "-scheme '$scheme' -configuration $configuration -destination 'id=$device' test" || failed=1
+        xc "-scheme '$scheme' -configuration $configuration -destination 'id=$device' -sdk $sdk test" || failed=1
```

Comment:
I guess that's why the destination specifier has a `platform` component. We could specify it instead, but it may be less convenient to conjure it up than the SDK name.